overlord/snapstate/conflict.go

```diff
@@ -157,6 +157,27 @@ func CheckChangeConflictRunExclusively(st *state.State, newChangeKind string) er
 	return checkChangeConflictExclusiveKinds(st, newChangeKind, "")
 }
 
+// isIrrelevantChange checks if a change is ready or it can be ignored
+// if matching the passed ID, for checking conflicts purposes.
```

Comment:
nitpick: this reads a bit better
```suggestion
// if matching the passed ID, for conflict checking purposes.
```